test/sanity/validate-modules/main.py

```diff
@@ -1294,7 +1297,8 @@ def _validate_argument_spec(self, docs, spec, kwargs):
                 self.reporter.error(
                     path=self.object_path,
                     code=326,
-                    msg=("argument_spec for '%s' defines choices as '%s' but documentation defines choices as '%s'" % (arg, arg_choices, doc_choices))
+                    msg=("Argument '%s' in argument_spec defines choices as '%r' "
```

Comment:
Remove the quotes from around `%r` as that may cause a something like `''foo''` to appear in the output.

Reply:
Fixed.